package.json

```diff
@@ -35,22 +35,22 @@
   },
   "homepage": "https://github.com/dabapps/create-webpack-config#readme",
   "dependencies": {
-    "@babel/core": "^7.12.3",
+    "@babel/core": "^7.12.9",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-react": "^7.12.1",
-    "@babel/preset-typescript": "^7.12.1",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-const-enum": "0.0.5",
+    "@babel/preset-env": "^7.12.7",
+    "@babel/preset-react": "^7.12.7",
+    "@babel/preset-typescript": "^7.12.7",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-const-enum": "^1.0.1",
```

Comment:
big version jump on this but I'm guessing they just actually went 1.0 rather than lots of breaking changes?

Reply:
No idea. 😂 it'll probably work - has no config, so nothing for us to update. I assume they've just fixed a bunch of bugs and are now happy with it.

Reply:
(they have no release notes or change log)

Reply:
okey doke - that sounds a little worrying that they dont but also if we don't interact via a config then also it should still work. I guess you'll find out when you test it anyway :)